app/src/main/java/org/simple/clinic/registration/name/RegistrationNameEffect.kt

```diff
@@ -5,3 +5,5 @@ import org.simple.clinic.user.OngoingRegistrationEntry
 sealed class RegistrationNameEffect
 
 data class PrefillFields(val entry: OngoingRegistrationEntry): RegistrationNameEffect()
+
+data class ValidatedEnteredName(val name: String): RegistrationNameEffect()
```

Comment:
Can we rename this class name please?
```suggestion
data class ValidateEnteredName(val name: String): RegistrationNameEffect()
```

Reply:
Good catch, done!